taas/mapping.py

```diff
@@ -0,0 +1,99 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Mapping(object):
+    """
+        This class handles mappings between our raw input, and what we generate.
```

Comment:
@Rtzq0 : Thank you! Do you know off the top of your head if there's an automated documentation linter that checks for this? I'm way better at writing code when I have a robot reminding me what to do. :)